lightningd/lightningd.c

```diff
@@ -174,7 +174,7 @@ static struct lightningd *new_lightningd(const tal_t *ctx)
 	 * book to hold all the entries (and trims as necessary), and multiple
 	 * log objects which each can write into it, each with a unique
 	 * prefix. */
-	ld->log_book = new_log_book(ld, 10*1024*1024);
+	ld->log_book = new_log_book(ld, 100*1024*1024);
```

Comment:
nit: Do you think that this is a revert commit of https://github.com/ElementsProject/lightning/commit/71381eb5be866f51b01b6d5f691999ee515edadc ? maybe we need just to update the comment?

Reply:
Whoops. Github did something terrible there. I intended for the comment to be updated; not the code.

Reply:
Not sure how that happened. Fixed the comment now as opposed to the code.

Reply:
>Not sure how that happened. Fixed the comment now as opposed to the code.
Cool!

Can squash the 2 commits in only one?

Reply:
done